src/styles/index.scss

```diff
@@ -2,2 +1,208 @@
+@import 'variables';
+@import 'stars';
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+// Header block
+
+.header {
+  width: 100%;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  &__container {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-inline: 50px;
+  }
+
+  &__logo-link {
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav {
+  height: 60px;
+  display: flex;
+
+  &__list {
+    padding: 0;
+    display: flex;
+    align-items: center;
+    list-style: none;
+  }
+
+  &__list-item .nav__item {
```

Comment:
Selectors with element and class combined like '.nav__list-item .nav__item' can be overly specific and hard to override later if needed. Consider simplifying the selector to just use the class name if it's unique enough.